Source/UINavigation/Private/UINavInputContainer.cpp

```diff
@@ -301,11 +301,9 @@ int UUINavInputContainer::GetOffsetFromTargetColumn(const bool bTop) const
 		case 2:
 			if (bTop) return (TargetColumn == ETargetColumn::Right);
 			else return -(TargetColumn != ETargetColumn::Right);
-			break;
```

Comment:
Why did you remove this? This will change the behavior of the switch block...

Reply:
this is correct, it doesn't change the behavior or Switch. I've removed it as my compiler keep complaining about unreachable code.

`if return and else return`
that means that the compiler will never reach the break as in all the case it returns before

The structure of a switch is:
Switch(v)
case X:
case Y:
etc...